src/hooks/useDrops.ts

```diff
@@ -14,7 +14,7 @@ const useDrops = ({
   ctx: MoonstreamWeb3ProviderInterface
 }) => {
   const [claimsPage, setClaimsPage] = React.useState(0)
-  const [claimsPageSize, setClaimsPageSize] = React.useState(5)
+  const [claimsPageSize, setClaimsPageSize] = React.useState(500)
```

Comment:
Was this intentional? Seems like a very large default page size.

Reply:
rn it's should be big enough because we needs all list. If we display whole list of claims (to make search works for example) then we need all file. 500 is the limit when we fetching all data. I'll change this logic but maybe I'll need some help from backend